dendron-main.code-workspace

```diff
@@ -49,6 +49,9 @@
         {
             "path": "packages/nextjs-template"
         },
+        {
+            "path": "packages/dendron-plugin-views"
```

Comment:
plugin-views or plugin-webviews?

Reply:
just left it as plugin views since we don't have other sort of views besides webviews